src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.css

```
@@ -16,6 +16,20 @@
   flex-direction: column;
 }
 
+.searchInputsRow {
+  display: grid;
+
+  /* 2 columns */
+  grid-template-columns: 1fr auto;
+  column-gap: 16px;
+
+  @media only screen and (max-width: 800px) {
+    /* Switch to 2 rows from 2 columns */
+    grid-template-columns: auto;
+    grid-template-rows: auto auto;
+  }
+}
+
```

This is a CSS file, so you can't nest stuff, please rewrite this to be normal CSS.

```suggestion
.searchInputsRow {
  display: grid;

  /* 2 columns */
  grid-template-columns: 1fr auto;
  column-gap: 16px;
}

@media only screen and (max-width: 800px) {
  .searchInputsRow {
    /* Switch to 2 rows from 2 columns */
    grid-template-columns: auto;
    grid-template-rows: auto auto;
  }
}

```